src/pages/_app.js

```diff
@@ -296,7 +296,11 @@ export default class App extends BaseApp {
 
     const availableLocales: Locale[] = Object.keys(allTranslations).map(localeFromString);
 
-    const appContext = { baseUrl };
+    const appContext = {
+      baseUrl,
+      categories: appProps.categories,
+      // TODO add joined mapping event here
```

Comment:
+0: In my current implementation of joined events I'm passing the joined event down the tree. I'm not convinced yet if we should add it to the global `AppContext`. #globalsphobia

Reply:
Otherwise I will have to pass it through
NodeToolbarFeatureLoader -> NodeToolbar ->  WheelchairStatusEditor -> RadioStatusEditor just for the purpose of sending a tracking event. 

This feels a bit like polluting the hierarchy too much. I could also move the tracking logic into eg. the app, so that I would have all the data available there, no need for context then.

Any opinions? Also @opyh potentially?

Reply:
I think it's okay to have this there globally as it's one of the few things that are actually a global context affecting everything in the app - on the same level as a potential user auth session.

Reply:
Sure. I agree so far. Only thing we need to consider is that we only get the joined mapping event id pretty late (only when we are client-side and therefore local storage is available). So I guess `_app.js` might not be the right place to set it.

Reply:
Good point, will consider this when going on with this.